tag/nitag.yml

```diff
@@ -0,0 +1,1749 @@
+      operationId: GetTagMinimum
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+      responses:
+        200:
+          description: Minimum aggregate value
+          schema:
+            type: string
+        204:
+          description: No aggregate value found
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/tags/{path}/values/aggregates/max:
+    get:
+      tags:
+        - tag aggregates
+      summary: Get the value of a tag's "max" aggregate
+      description: Get the value of a tag's "max" aggregate
+      operationId: GetTagMaxAggregate
+      responses:
+        200:
+          description: Maximum aggregate value
+          schema:
+            type: string
+        204:
+          description: No aggregate value found
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+  /v2/tags/{path}/values/aggregates/mean:
+    get:
+      tags:
+        - tag aggregates
+      summary: Get the value of a tag's "avg" aggregate
+      description: Get the value of a tag's "avg" aggregate
+      operationId: GetTagMeanAggregate
+      responses:
+        200:
+          description: Mean aggregate value
+          schema:
+            type: string
+        204:
+          description: No aggregate value found
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+  /v2/tags/{path}/values/aggregates/count:
+    get:
+      tags:
+        - tag aggregates
+      summary: Get the value of a tag's "count" aggregate
+      description: Get the value of a tag's "count" aggregate
+      operationId: GetTagCountAggregate
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+      responses:
+        200:
+          description: Count aggregate value
+          schema:
+            type: string
+        204:
+          description: No aggregate value found
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/tags/{path}/reset-aggregates:
+    post:
+      tags:
+        - tag aggregates
+      summary: Reset the aggregates of a tag
+      description: Resets aggregate values for a tag.
+      operationId: ResetTagAggregates
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+      responses:
+        200:
```

Comment:
204.

Reply:
Cloud returns a 200, but server returns a 204. For all of these cases, I'll file a CAR later.